src/common/dout.h

```diff
@@ -69,6 +76,31 @@ struct is_dynamic<dynamic_marker_t<T>> : public std::true_type {};
 // generic macros
 #define dout_prefix *_dout
 
+#ifdef WITH_SEASTAR
+#define dout_impl(cct, sub, v)                                          \
+  do {                                                                  \
+    if (ceph::common::local_conf()->subsys.should_gather(sub, v)) {     \
```

Comment:
is it `ceph::common::get_config()` rather than `ceph::common::local_conf()`?

Reply:
i don't think we have `ceph::common::get_config()`? could you please point me to it?

Reply:
Ahh, sorry... Just realized I was looking at the wrong function.